client/modules/wj-css/src/abstracts/_mixins.scss

```diff
@@ -19,3 +19,31 @@
     outline: none;
   }
 }
+
+/// Hides content visually but not to screen readers.
+@mixin hide-visually {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+  white-space: nowrap;
+  clip: rect(1px, 1px, 1px, 1px);
+  clip-path: inset(100%);
+}
```

Comment:
General question: Right now `[[invisible]]` just does `color: transparent`. How should this interact with this styling?

Reply:
This mixin is basically like a hide nuke - it does everything it can to visually hide the element. I assume invisible text is supposed to just be invisible, but take up room and affect layout.

Reply:
Oh right, okay.